weld-lite-extension-translator/src/main/java/org/jboss/weld/lite/extension/translator/MessagesImpl.java

```diff
@@ -4,83 +4,81 @@
 import jakarta.enterprise.inject.build.compatible.spi.Messages;
 import jakarta.enterprise.inject.build.compatible.spi.ObserverInfo;
 import jakarta.enterprise.lang.model.AnnotationTarget;
-
-import java.util.logging.Logger;
+import org.jboss.weld.exceptions.DeploymentException;
+import org.jboss.weld.lite.extension.translator.logging.LiteExtensionTranslatorLogger;
 
 class MessagesImpl implements Messages {
 
     private final String AT = " at ";
 
     private final SharedErrors errors;
-    private final Logger logger;
 
-    MessagesImpl(java.lang.reflect.Method method, SharedErrors errors) {
+    MessagesImpl(SharedErrors errors) {
         this.errors = errors;
-        this.logger = Logger.getLogger(method.getDeclaringClass().getName());
```

Comment:
This is a little unfortunate. Previously, each BCExtension got its own logger category (the extension class), but with this change, they are all lumped together (under the translator package).

Reply:
@Ladicek Does that make much difference it terms of the message user gets? Do you think we should somehow preserve the notion of declaring class the logged message?

Reply:
I don't think it's _that_ big of a deal, but AFAIK JBoss Logging would easily allow retaining the original behavior, which I'd find nice.

Reply:
Yea, I could leave this as it was but I wanted to align with the rest of the logging done in this module. Personally, I didn't see any difference in it...